src/components/Marketplace/Marketplace.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {Navigation} from "../index";
+import PTProfile from "../Profile/PTProfile";
+
+function Marketplace() {
```

Comment:
no functionality, maybe we should discuss what funct will look like in next meeting?